src/bun.js/bindings/helpers.cpp

```diff
@@ -1,6 +1,260 @@
 #include "root.h"
 #include "helpers.h"
 #include "BunClientData.h"
+#include <string.h>
+
+const char* errno_string(int errorno)
```

Comment:
Don't we already have all these strings in Zig? Wouldn't this be duplicating the strings in the binary? Couldn't we expose a function to get those strings instead?